src/Symfony/Bundle/FrameworkBundle/EventListener/SuggestMissingPackageSubscriber.php

```diff
@@ -33,7 +33,7 @@ final class SuggestMissingPackageSubscriber implements EventSubscriberInterface
             '_default' => ['Doctrine ORM', 'symfony/orm-pack'],
         ],
         'generate' => [
-            '_default' => ['SensioGeneratorBundle', 'sensio/generator-bundle'],
+            '_default' => ['MakerBundle', 'symfony/maker-bundle --dev'],
```

Comment:
This is the wrong suggestion IMO. Installing MakerBundle won't make the `generate:*` command work

Reply:
Sure, but that's a hint about the replacement. I'm fine with this.

Reply:
But the wording of the error won't tell you about that (it might tell you `You may be looking for a command provided by the "MakerBundle" which is currently not installed. Try running "composer require symfony/maker-bundle --dev".`, but you might have maker:bundle already installed.

If we want to suggest migrating from `generate:*` to `make:*` commands, we need a different suggestion system, not the suggestion meant to handle missing packages.

Reply:
perhaps keep pointing users to SensioGeneratorBundle, which on itself is deprecated anyway.

then remote the entry in 6.x

Reply:
Good point. Closing here, see #48455